kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -144,4 +162,20 @@
     margin-bottom: -6px;
   }
 
+  .loading {
+    animation: spin 2s infinite linear;
+  }
+
+  // Based on this css
+  // https://github.com/loadingio/loading.css/blob/master/dist/loading.css#L1643
```

Comment:
Nice library of loading animations. Let's merge this one and propose the other options to Joana.